### PR TITLE
New package: parprouted-0.7.

### PR DESCRIPTION
**Pull request by fosslinux**

### CODE REVIEW

**Comment by fosslinux** [comment]
Uh, didn't see #21196 but it is broken, it doesn't have the required iproute2 dep and this includes a service.

File: srcpkgs/parprouted/template [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+maintainer="fosslinux <fosslinux@aussies.space>"
+license="GPL-2.0-or-later"
+homepage="http://freecode.com/projects/parprouted"
+distfiles="http://www.hazard.maks.net/parprouted/parprouted-${version}.tar.gz"
+checksum=d63643c95d4149a3d71112da5892f459433456bf0279bfa304625f109178defc
+
+do_install() {
+	vbin "${pkgname}"
+	vman "${pkgname}.8"
+	vsv "${pkgname}"
```
Comment (Duncaen) [inline srcpkgs/parprouted/template:17]: Probably better to not use the pkgname variable here.

File: srcpkgs/parprouted/files/parprouted/conf [hunk 2]
```diff
@@ -0,0 +1,2 @@
+#EXIT_IF=wlan0
+#IN_IF=eth0
```
Comment (Duncaen) [inline srcpkgs/parprouted/files/parprouted/conf:2]: We usually don't ship conf files if we would do that then we would also have to mark them as configuration file.

File: srcpkgs/parprouted/files/parprouted/run [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+else
+    echo "Add conf file!"
+    exit 1
+fi
+
+if [ -z "${EXIT_IF}" ] || [ -z "${IN_IF}" ]; then
+    echo "Add data in conf file!"
+    exit 1
+fi
+
```
Comment (Duncaen) [inline srcpkgs/parprouted/files/parprouted/run:13]: Not a fan of this being part of the run script.
```
#!/bin/sh
[ -r ./conf ] && . ./conf
exec paperouted ${OPTS}
```

should be enough, let the service fail and don't add logic to the run script that might change.

**Review by Duncaen — CHANGES_REQUESTED** [review]

File: srcpkgs/parprouted/files/parprouted/run (hunk 3 above)
Comment (fosslinux) [reply srcpkgs/parprouted/files/parprouted/run:13]: yeah ok

**Comment by fosslinux** [comment]
All fixed up.

**Comment by fosslinux** [comment]
... other one got merged

**Pull request closed without merging**